dojo/templates/dojo/login.html

```diff
@@ -77,6 +77,14 @@ <h3>Login</h3>
                     </div>
                 {% endif %}
 
+                {% if KEYCLOAK_ENABLED is True %}
+                <div class="col-sm-offset-1 col-sm-2">
+                    <button class="btn btn-success" type="button">
+                        <a href="{% url 'social:begin' 'keycloak' %}?next={{ request.GET.next }}" style="color: rgb(255,255,255)">Login with Keycloak</a>
```

Comment:
Can you make button test customable  as is for SAML

Reply:
I have added an option in a similar fashion as the SAML button already has. 
Though I have to admit, in `context_processors.py` I wasn't exactly sure about the naming, it would be kind if you could take a look an see whether that is sensible/accebtable.